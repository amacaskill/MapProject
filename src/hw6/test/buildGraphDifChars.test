LoadGraph sameBook buildGraphDifChars.tsv

ListNodes sameBook

ListChildren sameBook char1
ListChildren sameBook char2 
ListChildren sameBook char3
ListChildren sameBook char4