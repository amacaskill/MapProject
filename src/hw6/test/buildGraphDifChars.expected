loaded graph sameBook

sameBook contains: char1 char2 char3 char4

the children of char1 in sameBook are: char2(book1) char3(book1) char4(book1)
the children of char2 in sameBook are: char1(book1) char3(book1) char4(book1)
the children of char3 in sameBook are: char1(book1) char2(book1) char4(book1)
the children of char4 in sameBook are: char1(book1) char2(book1) char3(book1)